Gems/WarehouseAutomation/gem.json

```diff
@@ -22,10 +22,10 @@
         "Linux"
     ],
     "icon_path": "preview.png",
-    "requirements": "This gem requires ROS 2 gem and PhysX gem.",
+    "requirements": "This gem requires ROS 2 gem and PhysX5 gem.",
     "dependencies": [
         "ROS2==3.0.0",
-        "PhysX"
+        "PhysX5"
```

Comment:
ditto here, we should update `compatible_engines` as that will let users see in the Project Manager that this gem is not compatible with older versions of the engine

Reply:
also we should bump the gem version to `1.1.0`